### DISCUSSION TITLE
如何禁用2FA验证？

### DISCUSSION THREAD

**Discussion by WindowsRegedit** [AlistGo/alist#1795] [Q&A] [answered]
因为我最近重置了手机，Google Authenticator的验证码没了，所以有没有办法在不重装Alist，并且无法登录的情况下禁用2FA验证？感谢！

**Comment by loryncien** [comment, marked as answer]
进入程序目录后运行即可看见帮助说明，如docker部署，docker exec -it alist ./alist
具体删除是docker exec -it alist ./alist cancel2fa
